community/samples/serving/helloworld-java-quarkus/.dockerignore

```diff
@@ -0,0 +1,10 @@
+README.md
```

Comment:
Should the _index.md also be listed here?

Reply:
not critical, but added.